### ISSUE TITLE
Definition for `eslint-plugin-jsx-a11y` not found

### ISSUE THREAD

**Issue by kyleholzinger** [codeclimate/codeclimate-eslint#271]
I'm using the eslint plugin `eslint-plugin-jsx-a11y` and I noticed that in one of my recent builds there were a bunch of errors saying:
<img width="541" alt="screen shot 2017-05-07 at 2 29 54 pm" src="https://cloud.githubusercontent.com/assets/2652762/25783883/ac013dea-3331-11e7-83b9-e61c7c0a0593.png">

Is this still supported on codeclimate-eslint? I checked [in the docs](https://docs.codeclimate.com/v1.0/docs/eslint) and it looks like it's still in there.

Here's the config I'm using:
```yml
eslint:
  enabled: true
  channel: "eslint-3"
```

**Comment by chrishulton** [comment]
Hi @kyleholzinger, thanks for reporting the problem. It looks like our `eslint-plugin-jsx-a11y` got bumped from 4.0.0 to 5.0.0 in a recent maintenance commit: https://github.com/codeclimate/codeclimate-eslint/commit/ed5cf15232997d6c99d69b2c8e2570e633e76a1e

Based on [their changelog](https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/v5.0.0), it looks like that rule got refactored in a breaking change to a more generic`alt-text`.

I think updating the rule from `jsx-a11y/img-has-alt` to `jsx-a11y/alt-text` should fix the issue. You can check out [their rule docs](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/v5.0.0/docs/rules/alt-text.md) to see if any of the rule configuration also needs to change.

**Comment by kyleholzinger** [comment]
Thanks for the info @chrishulton. Unfortunately, we're not even using the `img-has-alt`, I'm pretty sure it's in one of the dependencies. Here is our `.eslintrc.json`:
```json
{
  "env": {
    "browser": true,
    "es6": true
  },
  "plugins": ["mocha"],
  "extends": "airbnb",
  "rules": {
    "indent": [
      2,
      2,
      {
        "MemberExpression": 1,
        "CallExpression": {
          "arguments": "first"
        }
      }
    ],
    "linebreak-style": [
      2,
      "unix"
    ],
    "no-multiple-empty-lines": [
      2,
      {"max": 1}
    ],
    "quotes": [
      2,
      "double"
    ],
    "semi": [
      2,
      "always"
    ],
    "key-spacing": [
      2,
      {
        "beforeColon": false,
        "afterColon": true
      }
    ],
    "space-infix-ops": [
      2
    ],
    "no-var": [
      1
    ],
    "mocha/no-exclusive-tests": "error",
    "no-underscore-dangle": "off",
    "arrow-body-style": "off",
    "class-methods-use-this": "off",
    "consistent-return": "off",
    "func-names": "off",
    "space-before-function-paren": "off",
    "no-void": "off",
    "no-mixed-operators": "off",
    "no-return-assign": "off",
    "no-restricted-syntax": "off",
    "no-param-reassign": "off",
    "newline-per-chained-call": "off",
    "max-len": "off",
    "no-plusplus": "off",
    "no-unused-expressions": "off",
    "no-multi-assign": "off",
    "no-cond-assign": "off",
    "camelcase": "off",
    "eqeqeq": "off",
    "radix": "off",
    "one-var": "off",
    "no-restricted-properties": "off",
    "no-nested-ternary": "off",
    "no-use-before-define": "off",
    "no-empty-function": "off",
    "no-useless-constructor": "off",
    "no-useless-escape": "off",
    "default-case": "off",
    "no-shadow": "off",
    "prefer-rest-params": "off",
    "prefer-spread": "off",
    "no-alert": "off",
    "prefer-const": "off",
    "no-new": "off"
  }
}
```
And the `eslint-plugin-mocha` does not use that rule

**Comment by kyleholzinger** [comment]
[Found it](https://github.com/airbnb/javascript/commit/01e39775972e3457defd64af2e94843a848d4fe2). Looks like the latest version of the airbnb config does not work with `jxs-a11y`'s new version.

**Comment by chrishulton** [comment]
Ah, good find.  I think it makes sense for us to revert that upgrade and go back to 4.0.0 until airbnb config gets a corresponding update.  I'll follow up on this tomorrow.

**Comment by kyleholzinger** [comment]
👍👍 sounds like a safe bet! Thanks for getting back to me so quickly

**Comment by chrishulton** [comment]
Merged in the downgrade of `jsx-a11y` to `4.0.0`. This should be resolved now.

**Comment by kyleholzinger** [comment]
Awesome! All good from my end 🎉
